deployment/helm/friend-service/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.0.1-SNAPSHOT"
+description: A Helm chart for event-sourcing-microservices-example friend service
```

Comment:
Let's change the description to something a bit more generic. "Helm chart for the friend microservice in hyperskale/social-network-example"